Assets/LeapC/LeapC.cs

```diff
@@ -366,33 +366,49 @@ public Leap.Vector ToLeapVector()
     {
       return new Leap.Vector(x, y, z);
     }
+
+    public UnityEngine.Vector3 ToUnityVector()
```

Comment:
Looks like these somehow made it into LeapC.  I don't think they are supposed to be here, since they reference UnityEngine.  We might want to add some extension methods for conversion (for Quaternion too)